programs/wen_new_standard/src/instructions/mint/royalties/add.rs

```diff
@@ -43,8 +43,11 @@ pub struct AddRoyalties<'info> {
     pub mint: Box<InterfaceAccount<'info, Mint>>,
     /// CHECK: This account's data is a buffer of TLV data
     #[account(
+        init,
```

Comment:
extra metas is only needed if we add transfer hook so you're moving it from create mint to here?